pramen/core/src/main/scala/za/co/absa/pramen/core/runner/PipelineSparkSessionBuilder.scala

```diff
@@ -64,7 +64,16 @@ object PipelineSparkSessionBuilder {
       case (builder, (key, value)) => builder.config(key, value)
     }
 
-    val spark = sparkSessionBuilderWithExtraOptApplied.getOrCreate()
+    val isHiveSupported = extraOptions.exists(_._1.contains("hive"))
```

Comment:
Sorry for my terrible nitpicking but just thinking if `isHiveSupportRequired` or `requiresHiveSupport` (or something that implies we need hive support) wouldn't be a better name.